modules/mobile/spec/request/appointments_create_request_spec.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
```

Comment:
I renamed this file because it was named to be specific to requests but the actual tests within it were for both proposed and booked.